src/helpers/file.ts

```diff
@@ -0,0 +1,7 @@
+import * as shell from 'shelljs';
+
+export const readConfigFile = () => {
```

Comment:
Why you do `variable = () => { ...}`?

Reply:
It's an arrow-function, not a variable, you can check the definition [here](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Functions/Arrow_functions).